index.js

```diff
@@ -39,6 +39,10 @@ function parse(str) {
   var hasBlob = seg[2] === 'blob';
   if (hasBlob && !isChecksum(seg[3])) {
     obj.branch = seg[3];
+    if(seg.length > 4)
```

Comment:
thanks for the pr! would you mind updating code formatting to be consistent with the rest of the code? In particular the braces and whitespace around `if` statements. thanks!